src/seedu/addressbook/data/person/Block.java

```diff
@@ -0,0 +1,26 @@
+package seedu.addressbook.data.person;
+
+/**
+ * Represents a Person's block in his address.
+ */
+public class Block {
+    
+    private final String block;
```

Comment:
why is block final? Can the block value change? if the value of block changes and you declare the block variable final , without a setter method how can we change it? Check it out!